public/src/components/channelManagement/supportLandingPage/landingPageTestEditorTargetRegionsSelector.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+import { Checkbox, FormControlLabel, FormGroup, Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { Region, regionIds, regions } from '../../../utils/models';
+import { Targeting } from '../../../models/supportLandingPage';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  indentedContainer: {
+    marginLeft: spacing(3),
+  },
+}));
+
+interface LandingPageTestEditorTargetRegionsSelectorProps {
+  targeting: Targeting;
+  onTargetingUpdate: (targeting: Targeting) => void;
+  isDisabled: boolean;
+}
+
+const LandingPageTestEditorTargetRegionsSelector: React.FC<LandingPageTestEditorTargetRegionsSelectorProps> = ({
```

Comment:
Let's change the model slightly rather than duplicate the region selector component like this.
How about:
```
export interface SupportLandingPageTestTargeting {
  regionTargeting: RegionTargeting;
}

export interface SupportLandingPageTest extends Test {
  name: string;
  ...
  targeting: SupportLandingPageTestTargeting;
  ...
}
```
Then we can reuse the `TestEditorTargetRegionsSelector`

Reply:
yes, changed this